.github/workflows/nuitka-standalone-zip.yml

```
@@ -24,15 +41,91 @@ jobs:
+        shell: bash
+        run: |
+          version="$(ffmpeg -version | head -n1 | awk '{print $3}')"
+          echo "version=$version" >> "$GITHUB_OUTPUT"
+
+      - name: Cache bundled FFmpeg (Linux)
+        if: matrix.os == 'ubuntu-latest'
+        id: cache-ffmpeg-linux
+        uses: actions/cache@v4
+        with:
+          path: vendor/ffmpeg/linux
+          key: ffmpeg-nuitka-${{ matrix.os }}-${{ runner.image_version }}-${{ steps.ffmpeg-version-linux.outputs.version }}-bundle
```

actionlint (1.7.12)

[error] property "image_version" is not defined in object type {arch: string; debug: string; environment: string; name: string; os: string; temp: string; tool_cache: string}

(expression)